components/ai_chat/resources/untrusted_conversation_frame/components/markdown_renderer/index.tsx

```diff
@@ -98,6 +98,10 @@ export default function MarkdownRenderer(mainProps: MarkdownRendererProps) {
     return transformer
   }, [])
 
+  const removeReasoning = (text: string) => {
```

Comment:
nit: lift this function out of the component

Reply:
Moved out of `component`